test/blackbox-tests/test-cases/dune-package.t/run.t

```diff
@@ -1,12 +1,16 @@
-  $ dune build
-  $ dune_cmd cat _build/install/default/lib/a/dune-package | sed "s/(lang dune .*)/(lang dune <version>)/" | dune_cmd sanitize
+  $ mkdir a
+  $ cp dune dune-project a.opam a
+
+  $ dune build --root=a
+  Entering directory 'a'
+  $ dune_cmd cat a/_build/install/default/lib/a/dune-package | sed "s/(lang dune .*)/(lang dune <version>)/" | dune_cmd sanitize
   (lang dune <version>)
   (name a)
   (sections
    (lib
-    $TESTCASE_ROOT/_build/install/default/lib/a)
+    $TESTCASE_ROOT/a/_build/install/default/lib/a)
```

Comment:
I still don't see why are we using an absolute path here. We should be using a path relative to the dune-package file like we do every other path in dune-project.